astronomer/providers/amazon/aws/example_dags/example_sagemaker.py

```diff
@@ -46,6 +66,19 @@ def set_up(role_arn: str) -> None:
     raw_data_s3_key_input = "preprocessing/input/validation.csv"
     clean_data = "preprocessing/output/clean_data.csv"
 
+    # Training configs
+    training_job_name = f"provider-train-{str(uuid.uuid4())[:8]}"
+    train_data_csv = "train-processed-input-data/train.csv"
+    training_output_s3_key = "train-processed-output"
+
+    # model
+    model_name = "provider-KNN-model"
+
+    # Transform configs
+    transform_job_name = "provider-transform"
+    transform_data_csv = "transform-processed-input-data/transform.csv"
+    prediction_output_s3_key = "/transform"
+
     ecr_repository_image_uri = (
         f"{ACCOUNT_ID}.dkr.ecr.us-east-2.amazonaws.com/providers_repo:sagemaker-processing"
```

Comment:
Use AWS_DEFAULT_REGION instead of hardcoding us-east-2

Reply:
This ECR image is created in region 2 and @rajaths010494 is using this URI

Reply:
Fixed it